Specify correct MSRV

README says that the minimum supported Rust version is 1.42, but it seems incorrect.

https://github.com/Keats/validator/blob/bd5293e5abaa76f20b3e6c2f1fe262b1a1347981/README.md?plain=1#L27

```console
$ rustc --version; cargo version
rustc 1.42.0 (b8cedc004 2020-03-09)
cargo 1.42.0 (86334295e 2020-01-31)
```

```console
$ git rev-parse HEAD; git rev-parse --abbrev-ref HEAD
bd5293e5abaa76f20b3e6c2f1fe262b1a1347981
master
```

```console
$ cargo build
error: failed to parse manifest at `/home/naskya/workspace/validator/validator/Cargo.toml`

Caused by:
  failed to parse the `edition` key

Caused by:
  supported edition values are `2015` or `2018`, but `2021` is unknown
```

I think the correct MSRV is 1.70.0, because you can compile this crate with Rust 1.70.0 but not with 1.69.x.

Some additional info:

* Rust 2021 edition is stabilized in Rust 1.56.0 [(ref)](https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html#whats-in-1560-stable). 
* `once_cell 1.19.0` requires Rust 1.60 [(ref)](https://github.com/matklad/once_cell/blob/c48d3c2c01de926228aea2ac1d03672b4ce160c1/Cargo.toml#L7),
* `yoke 0.7.4`, `litemap 0.7.3`, and `icu_properties 1.5.1` requires Rust 1.67.
* I tried to compile (`cargo build --locked`) this crate with Rust 1.67.0 but got the following error
    ```console
    error[E0658]: use of unstable library feature 'once_cell'
     --> validator/src/validation/regex.rs:2:5
      |
    2 | use std::cell::OnceCell;
      |     ^^^^^^^^^^^^^^^^^^^
      |
      = note: see issue #74465 <https://github.com/rust-lang/rust/issues/74465> for more information
    ```
    * This feature is stabilized in Rust 1.70.0 [(ref)](https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html#oncecell-and-oncelock)

So I suggest that you add

```toml
rust-version = "1.70.0"
```

in `validator/Cargo.toml` and update the documentation.

Moreover, it would be nice if the MSRV is automatically checked by CI (I think you can `rustup default 1.70.0 && cargo fetch --locked && cargo check --frozen --all-features` to check the compatibility), given that MSRV may be (unconsciously) bumped by dependency updates.

Anyway, thanks a bunch for maintaining this crate!

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,12 +1,9 @@
 ## Changelog
 
-## 0.19 (unreleased)
+## 0.19 (2024/11/01)
 
 - Swap to using proc-macro-error-2 instead of proc-macro-error for Syn
 - Bumped MSRV to 1.81 because of error naming changes.
-
-## 0.18.2 (unreleased)
-
 - Add more ValidateRegex impl
 
 

--- a/README.md
+++ b/README.md
@@ -24,13 +24,13 @@
 Macros 1.1 custom derive to simplify struct validation inspired by [marshmallow](http://marshmallow.readthedocs.io/en/latest/) and
 [Django validators](https://docs.djangoproject.com/en/1.10/ref/validators/).
 
-The minimum supported version is Rust 1.42.
+The minimum supported version is Rust 1.70.
 
 Installation:
 
 ```toml
 [dependencies]
-validator = { version = "0.18", features = ["derive"] }
+validator = { version = "0.19", features = ["derive"] }
 ```
 
 A short example:

--- a/validator/Cargo.toml
+++ b/validator/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "validator"
-version = "0.18.2"
+version = "0.19.0"
 authors = ["Vincent Prouillet <hello@vincentprouillet.com"]
 license = "MIT"
 description = "Common validation functions (email, url, length, ...) and trait - to be used with `validator_derive`"
@@ -18,7 +18,7 @@ idna = "1"
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
-validator_derive = { version = "0.18", path = "../validator_derive", optional = true }
+validator_derive = { version = "0.19", path = "../validator_derive", optional = true }
 card-validate = { version = "2.3", optional = true }
 unic-ucd-common = { version = "0.9", optional = true }
 indexmap = { version = "2.0.0", features = ["serde"], optional = true }

--- a/validator_derive/Cargo.toml
+++ b/validator_derive/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "validator_derive"
-version = "0.18.2"
+version = "0.19.0"
 authors = ["Vincent Prouillet <hello@vincentprouillet.com"]
 license = "MIT"
 description = "Macros 1.1 implementation of #[derive(Validate)]"

--- a/validator_derive_tests/Cargo.toml
+++ b/validator_derive_tests/Cargo.toml
@@ -10,7 +10,7 @@ test_ui = []
 nightly = []
 
 [dev-dependencies]
-validator = { version = "0.18", path = "../validator", features = [
+validator = { version = "0.19", path = "../validator", features = [
     "card",
     "unic",
     "derive",

